packages/lesswrong/components/votes/OverallVoteAxis.tsx

```diff
@@ -133,20 +134,23 @@ const OverallVoteAxis = ({
     >
       {children}
     </LWTooltip>
-  )
-  const TooltipIfEnabled = (canVote
+  , [canVote, karmaTooltipTitle, whyYouCantVote, classes.tooltip, LWTooltip])
+  
+  const TooltipIfEnabled = React.useMemo(() => canVote
     ? ({children, ...props}: React.ComponentProps<typeof LWTooltip>) =>
       <LWTooltip {...props} popperClassName={classes.tooltip}>
         {children}
       </LWTooltip>
     : ({children}: {children: React.ReactNode}) => <>{children}</>
-  );
+  , [canVote, LWTooltip, classes.tooltip])
+
+
+  // Moved down here to allow for useMemo hooks
```

Comment:
Is this a comment from an LLM?  (Also, is that why everything is wrapped in `useMemo`?)

Reply:
useMemo's were done by Oli. Without them, the component rerenders while you're trying to hold down to do a strong vote.